src/twisted/spread/jelly.py

```diff
@@ -86,19 +86,6 @@
 _SetTypes = [set]
 _ImmutableSetTypes = [frozenset]
```

Comment:
I'd be tempted to remove these too

Reply:
I considered that, or perhaps changing them to tuples at least. I'm on the fence about whether it's likely that we'll want some other classes in there in the future, but perhaps simplicity now is a better option